packages/core/src/common/commands-common.ts

```diff
@@ -9,6 +9,16 @@ import { MAIN_MENU_BAR, MenuContribution, MenuModelRegistry } from './menu';
 import { CommandContribution, CommandRegistry } from './command';
 import { injectable } from "inversify";
 
+const enjson = require('./i18n/en.json');
+const esjson = require('./i18n/es.json');
+const frjson = require('./i18n/fr.json');
+const Globalize = require("globalize");
+
+// Load potential languages
+Globalize.loadMessages(enjson);
+Globalize.loadMessages(esjson);
+Globalize.loadMessages(frjson);
```

Comment:
Yes the problem is loading those files... using a library the require context will be that library path and so unless you have that library copy & pasted in each package you can't use require with the proper path.

I think having a few require lines per language in each package and a library to load these objects should be ok ?